frontend/public/style/_overrides.scss

```diff
@@ -4,16 +4,48 @@ form.pf-c-form {
   --pf-c-form__helper-text--Color: var(--pf-global--Color--200);
 }
 
-// Positions the img in OperatorHub that are nested with <span class="catalog-tile-pf-icon"><img ...>
-.catalog-tile-pf-icon--align-top {
-  vertical-align: top;
-}
-
 // Until Patternfly-React-Extensions is updated: https://github.com/patternfly/patternfly-react/issues/1146
 .catalog-tile-pf-title {
   @include co-break-word;
 }
 
+.co-catalog-tile {
+  .catalog-tile-pf-icon {
+    font-size: unset !important;
+  }
+  .co-catalog--logo {
+    height: 100%;
+    max-width: 80px;
```

Comment:
Might consider bumping this to around 100, some of the logos are a bit hard to see.